S3/English/ASS_27.ass

```diff
@@ -0,0 +1,97 @@
+﻿[Script Info]
+; Script generated by Aegisub 3.2.2
+; http://www.aegisub.org/
+ScriptType: v4.00+
+WrapStyle: 0
+ScaledBorderAndShadow: yes
+YCbCr Matrix: TV.601
+PlayResX: 1920
+PlayResY: 1080
+
+[Aegisub Project Garbage]
+Audio File: 万圣街_1920x1080_万圣街_27.mp4
+Video File: 万圣街_1920x1080_万圣街_27.mp4
+Video AR Mode: 4
+Video AR Value: 1.777778
+Video Zoom Percent: 0.375000
+Scroll Position: 51
+Active Line: 40
+Video Position: 4066
+
+[V4+ Styles]
+Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
+Style: Default,Arial,45,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,18,0,20,1
+Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,10,10,280,1
+
+[Events]
+Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+Dialogue: 0,0:01:02.51,0:01:05.56,Title,,0,0,0,, {\pos(970,920)} {\b1\an5\pos(970,880)\fs120\frz4.5\t(0,100,1.5\fs45)}Episode 27:  {\c&H7990E7&}Panic as the house get crowded{\b0}
+Dialogue: 0,0:01:06.75,0:01:08.35,Default,,0,0,0,,This is a scene from "{\i1}A song of blood and brain{\i0}"
+Dialogue: 0,0:01:08.60,0:01:09.92,Default,,0,0,0,,A play with the vampire queen
+Dialogue: 0,0:01:10.82,0:01:11.82,Default,,0,0,0,,This is "{\i1}The last days{\i0}"
+Dialogue: 0,0:01:12.17,0:01:13.97,Default,,0,0,0,,It's a climax part I acted
+Dialogue: 0,0:01:14.06,0:01:14.84,Default,,0,0,0,,It's so amazing
+Dialogue: 0,0:01:16.88,0:01:19.72,Default,,0,0,0,,This video is the part I had a line in "{\i1}The bad omen{\i0}"
```

Comment:
I hope my translations for the movie names are ok, and does this need all upper case?

Reply:
the TLs seem good, as for caps there are guidelines which normally gives "A Song of Blood anf Brain", "The Last Day" and "The Bad Omen"